pkg/lock/client.go

```diff
@@ -0,0 +1,35 @@
+package lock
+
+import (
+	"sync"
+	"time"
+
+	"github.com/cozy/cozy-stack/pkg/config"
+	"github.com/go-redis/redis"
+)
+
+type subRedisInterface interface {
+	SetNX(key string, value interface{}, expiration time.Duration) *redis.BoolCmd
+	Eval(script string, keys []string, args ...interface{}) *redis.Cmd
+}
+
+var mu sync.Mutex
+var globalRedisClient subRedisInterface
```

Comment:
It is not possible as the config is not loaded in init. 
To be included in the RedisClient refactoring card.